### ISSUE TITLE
Feature: Aggregate LNS logs by device on the portal

### ISSUE THREAD

**Issue by kbeaugrand** [CGI-FR/IoT-Hub-Portal#1605]
**Is your feature request related to a problem? Please describe.**
When I'm deploying a device, I'd like to have information about : 
- Is my device trying to connect?
- Is my device sending some message?
- Is the LNS authenticating the device well?
- ... 

**Describe the solution you'd like**
In addition to LSNR and RSSI data that we already have, it is useful to get LNS logs per devices and provide the user a way to see these logs on the portail in order to diagnose some connection issues... 

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by kbeaugrand** [comment]
@adcoly @Metal-Mighty, we discussed about that but as fare I know you didn't create an issue for that?

**Comment by Metal-Mighty** [comment]
I don't think we ever created an issue about that indeed. We have partially implemented it in our stack with contextual information added to the logs of our IoT Edge modules, but we currently don't have these information in the LNS. I'd be glad to discuss it with you over the next week !

**Comment by kbeaugrand** [comment]
This issue will be discussed this week during our PBR, don't hesitate to join us at this moment ;)

**Comment by kbeaugrand** [comment]
* We will take messages coming from the LoRaWanNetworkSrvModule.
The events will be filtered by the sender id (using the convention ``device-id``/``module-id``.
* We will assume by default that module-id is ``LoRaWanNetworkSrvModule`` and will let it configurable.
* We will keep by default 100 event logs per device and let it also configurable.

During the implementation, we will take the same job that is actually reading events from IoT Hub (same consumer group).

__Attention__
In the logs it could have some secrets (the auth code to the Azure Function), take it in account during the documentation, to provide insights to the users and encourage to use v2.2 of the starterkit that might have fixed that)
